### PR TITLE
Global Admin User Tests CRASM-3184

### PR DESCRIPTION
**Pull request by jyang2139**
## 🗣 Description ##

This PR adds Playwright tests for the Global Admin role, validating end-to-end navigation, permissions, and filter behavior across the dashboard. Tests confirm Global Admins can use everything including the navigation to reach VSDashboard, Findings Library / Inventory, Account Settings, and expand Admin Hub, Learning Center, and Support. Filters on VSDashboard and Findings Library allow selecting any Region and Organization. Sould have access to global admin restricted paths like /admin-tools and global-admin-only endpoints (e.g., /metrics/customers) are blocked. An axe a11y test suite checks for serious/critical violations on critical pages. These tests ensure the Global Admin experience reflects intended permissions and that all access is provided to the user. 

## 💭 Motivation and context ##

Tests for Global Admin Users ensure correct permissions — verifying the user has access to everything, preventing privilege bugs, and maintaining security compliance.

## 🧪 Testing ##

tested locally 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.

### CODE REVIEW

File: playwright/e2e/global-admin/global-admin.spec.ts [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+    const chosenRegion = await selectFromAutocomplete(
+      page,
+      /^region$/i,
+      /Region\s*2/i
+    );
+    await page.waitForTimeout(400);
+    const chosenOrg = await selectAnyOrganization(page, /^organization$/i);
+    await page.waitForTimeout(400);
+    await closeFilterDrawer(page, VS, { assertHidden: true });
+    const urlHasFilters = await urlHasBothFilters(page, VS, 1500);
```
Comment (cduhn17) [inline playwright/e2e/global-admin/global-admin.spec.ts:526]: The use of arbitrary timeouts (1500ms) may cause intermittent failures. Consider waiting on known events

File: playwright/e2e/global-admin/global-admin.spec.ts [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+  selectAnyOrganization,
+  isVisible,
+  hasValue,
+  urlHasBothFilters,
+  escapeForTextSelector,
+  VS,
+  INV
+} from '../../utils/filters';
+
+const UUID_RX = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}';
```
Comment (cduhn17) [inline playwright/e2e/global-admin/global-admin.spec.ts]: Is this used anywhere else? Is so consider moving this to a constants file.

File: playwright/e2e/global-admin/global-admin.spec.ts [hunk 3]
```diff
@@ -0,0 +1,772 @@
...
+    }
+
+    const expectedItems = [
+      /admin tools/i,
+      /manage organizations/i,
+      /manage users/i,
+      /user registration/i
+    ];
+
+    for (const rx of expectedItems) {
```
Comment (cduhn17) [inline playwright/e2e/global-admin/global-admin.spec.ts:48]: Consider adding a few tests asserting accessibility of empty states (e.g., no domain details) to ensure accessible fallback experiences.

File: playwright/e2e/global-admin/global-admin.spec.ts (hunk 2 above)
Comment (jyang2139) [reply playwright/e2e/global-admin/global-admin.spec.ts]: Will do that.

File: playwright/e2e/global-admin/global-admin.spec.ts (hunk 1 above)
Comment (jyang2139) [reply playwright/e2e/global-admin/global-admin.spec.ts:526]: The filters are not able to be applied in the playwright tests unless there is a timeout. The timeouts are there so the filter affects will actually apply. Not sure if it is because of the dashboard itself or because it's how playwright tests function. Will create reserach ticket.

File: playwright/e2e/global-admin/global-admin.spec.ts (hunk 3 above)
Comment (jyang2139) [reply playwright/e2e/global-admin/global-admin.spec.ts:48]: We’re deferring empty-state tests because the app never hits “empty,” we can’t reliably fake it yet, and any tests now would just skip or fail.

File: playwright/e2e/global-admin/global-admin.spec.ts (hunk 2 above)
Comment (jyang2139) [reply playwright/e2e/global-admin/global-admin.spec.ts]: e99d3e65923f3ebe85fe5704f2d674bf06b6fb42

**Review by cduhn17 — APPROVED** [review]
LGTM - comments have been answered.

**Review by rapidray12 — APPROVED** [review]
The suite is large (~770 lines). Consider splitting it into thematic specs (e.g., global-admin-nav.spec.ts, global-admin-a11y.spec.ts) to improve maintainability and parallel execution.

Timeout consistency

A few tests use fixed waits (waitForTimeout(400) / 1500). It would be better to rely on event-based waits (e.g., waitForResponse, waitForSelector) to improve reliability.


Add a brief comment or README note describing how to run role-specific E2E tests (pageAsGlobalAdmin, pageAsRegionalAdmin, etc.), since not all contributors may know those fixtures exist.

**Comment by jyang2139** [comment]
Issue with timeout for filters covered in: https://maestro.dhs.gov/jira/browse/CRASM-3400
Currently the fitlers don't apply without adding a timeout and need to figure out why.
